README.md

```diff
@@ -17,32 +17,34 @@ The following steps should only be used to install if you are planning on contri
 1. Install [Node.js 16.x](https://nodejs.org/en/download/)
 1. Install [Rust](https://www.rust-lang.org/learn/get-started).
 1. Install [Yarn](https://classic.yarnpkg.com/en/docs/install).
+1. Run `yarn install` from the root directory to install packages (see sections below for OS specific details).
```

Comment:
Ahh it may :) My issue was that the OS specific stuff was sort of mixed together, but I also didn't want to move the install command to the bottom. I will think of a better way to format